lib/improver/ensemble_copula_coupling/ensemble_copula_coupling_constants.py

```diff
@@ -53,6 +53,7 @@
     "air_temperature": (
         Bounds((-40-ABSOLUTE_ZERO, 50-ABSOLUTE_ZERO), "Kelvin")),
     "wind_speed": Bounds((0, 50), "m s^-1"),
+    "wind_speed_of_gust": Bounds((0, 50), "m s^-1"),
```

Comment:
I do feel this should be higher, but I don't know by how much,

Reply:
The highest recorded UK gust is 150.3 knots (~77.3 m/s) according to https://www.metoffice.gov.uk/learning/wind/windiest-place-in-uk.

Reply:
We could bump it up to 100 m/s or something. Would give us some overhead. Do we need to consider global gusts or just the UK?

Reply:
I'd say change it to 70 or 80. We can always revisit all of these values at some point. For now I think all of these are only over the uk.